Documentation/guides/tracing.md

```diff
@@ -98,6 +98,63 @@ And the output files should be found in the current directory. You can
 use the `-o` switch if you would prefer to output them to a specific
 directory.
 
+## How to get GC memory dumps?
+
+If running on desktop, you can simply do:
```

Comment:
Would it be useful to explicitly mention that the `dotnet-gcdump` program must be separately installed?

```
$ dotnet tool install --global dotnet-gcdump
```

and furthermore that it's installed into `$HOME/.dotnet/tools`?

I tried running `dotnet gcdump --help`, and it immediately failed because I didn't have it installed:

```
% dotnet gcdump --help
Could not execute because the specified command or file was not found.
Possible reasons for this include:
  * You misspelled a built-in dotnet command.
  * You intended to execute a .NET program, but dotnet-gcdump does not exist.
  * You intended to run a global tool, but a dotnet-prefixed executable with this name could not be found on the PATH.
```